packages/shared/src/types/index.ts

```diff
@@ -274,6 +280,8 @@ export interface FrontMatterMeta {
   lineNumbers?: boolean;
   overviewHeaders?: number;
   titleSuffix?: string;
+  head?: [string, Record<string, string>][];
+  [key: string]: unknown;
```

Comment:
just for typing hint?

Reply:
Because the pages frontmatter can include any key-value, this type allows reading of any additional properties/keys that have not been defined in the `FrontMatterMeta`. It also aids into making the typing for `loadFrontmatter` work with generic.

Reply:
sounds great